.github/workflows/node.js.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: E-commerce App
+version: 1.0.0
+description: A simple Node.js application for E-commerce App
+author: Team predetor
+
+on:
+  push:
+    branches: [ "*" ]
```

Comment:
I believe having at least develop branch here is better

Reply:
done